src/js/views/patients/patient/patient.scss

```diff
@@ -16,7 +16,6 @@
   display: flex;
   flex: initial;
   min-height: 0;
-  width: 256px;
```

Comment:
Removing this and adding it to `.patient-sidebar` seems to fix the issue.

My understanding is that this `.patient__sidebar` class is only used for the patient sidebar on the patient dashboard. The other patient sidebars use different CSS classes. So it shouldn't effect any other sidebars.